docs/source/tutorial/typesfuns.rst

```diff
@@ -438,8 +436,8 @@ importing ``Data.Fin``, or can be declared as follows:
        FS : Fin k -> Fin (S k)
 
 From the signature,  we can see that this is a type constructor that takes a ``Nat``, and produces a type.
-So this is not a set in the sense of a collection that is a container of objects,
-rather it is the canonical set of unnamed elements, as in "the set of 5 elements," for example.
+So this is not a set in the sense of a collection that is a container of objects;
+rather, it is the canonical set of unnamed elements, as in “the set of 5 elements”, for example.
```

Comment:
Literal unicode quotes accidentally left in from editing on MacOS?

Reply:
There were already lots of unicode quotes, so I assumed that is the preferred way to represent them outside of code. Should I replace all existing ones?